js/foundation.dropdown.js

```diff
@@ -165,7 +166,7 @@
         param = (direction === 'top') ? 'height' : 'width',
         offset = (param === 'height') ? this.options.vOffset : this.options.hOffset;
 
-    if(($eleDims.width >= $eleDims.windowDims.width) || (!this.counter && !Foundation.Box.ImNotTouchingYou(this.$element))){
+    if(($eleDims.width >= $eleDims.windowDims.width) || (!this.counter && !Foundation.Box.ImNotTouchingYou(this.$element, null, true, false))){
```

Comment:
This is still a problem because of the repositioning. If the dropdown overflows with the top then it repositions itself and could overlap with the left/right. Tempted to remove.
```(!this.counter && !Foundation.Box.ImNotTouchingYou(this.$element, null, true, false)```
Not understanding when you would want to dynamically set the width except when:
```$eleDims.width >= $eleDims.windowDims.width```